hibernate-core/src/main/java/org/hibernate/event/spi/HibernateEvent.java

```diff
@@ -0,0 +1,13 @@
+/*
+ * Hibernate, Relational Persistence for Idiomatic Java
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later.
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.event.spi;
+
+import org.hibernate.Incubating;
+
+@Incubating
+public interface HibernateEvent {
```

Comment:
The name "event" is already overloaded - I'm not against reusing it but in that case at least add a little bit of context here, like what's the intention for this new interface.

Reply:
I agree about the naming.  IMO, `HibernateJfrEvent`, `HibernateMonitoringEvent`, ... is better

Reply:
renamed to `HibernateMonitoringEvent`